src/components/filters/OptionFilter.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { useMemo, useState } from 'react';
+import { Column } from '@tanstack/react-table';
+
+// components
+import MultiSelect from '../MultiSelect';
+
+export type Option = {
```

Comment:
- [ ] in the DateFilter component you have the same type, so move it to some shared interface to avoid duplicates